frontend/src/components/readsets/ReadsetMetricContent.tsx

```diff
@@ -0,0 +1,72 @@
+import { ExpandableConfig } from "antd/lib/table/interface"
+import { ObjectWithReadset } from "./ReadsetsTableColumns"
+import { ReactNode, useMemo } from "react"
+import { TableProps, Tooltip, Typography } from "antd"
+import { MinusCircleTwoTone, PlusCircleTwoTone } from "@ant-design/icons"
+import { Readset } from "../../models/frontend_models"
+import React from "react"
+
+export const ReadsetMetricContent = (): ExpandableConfig<ObjectWithReadset> => {
+    const columnTitle: ReactNode = <Typography.Text>Metrics</Typography.Text>
+    function checkIfDecimal(str: string) {
+      const num = parseFloat(str)
+      if (String(num).includes('.')) {
+          return num.toFixed(3)
+      } else {
+          return num
+      }
+    }
+    return {
```

Comment:
useMemo ?

Reply:
1. I dont think useMemo is necessary in this instance since it renders on a optionally filed variable (metric) with very few amounts of data in each.
2. I believe useMemo is to be used adequately, one of the key principles in of useMemo is to memoize (remember) the data for the next render, in this instance, i didn't see any substantial benefit, it wouldn't offer any increase in performance when it comes down to this component.